doc/release-notes.md

```diff
@@ -136,29 +138,40 @@ https://www.torproject.org/docs/tor-manual.html.en
 
 This allows running bitcoind without having to do any manual configuration.
 
-Relay: Any sequence of pushdatas in OP_RETURN outputs now allowed
------------------------------------------------------------------
+Relay and Mining: Any sequence of pushdatas in OP_RETURN outputs now allowed
+----------------------------------------------------------------------------
 
 Previously OP_RETURN outputs with a payload were only relayed and mined if they
 had a single pushdata. This restriction has been lifted to allow any
-combination of data pushes and numeric constant opcodes (OP_1 to OP_16). The
-limit on OP_RETURN output size is now applied to the entire serialized
-scriptPubKey, 83 bytes by default. (the previous 80 byte default plus three
-bytes overhead)
+combination of data pushes and numeric constant opcodes (OP_1 to OP_16) after
+the OP_RETURN. The limit on OP_RETURN output size is now applied to the entire
+serialized scriptPubKey, 83 bytes by default. (the previous 80 byte default plus
+three bytes overhead)
 
-Relay: Priority transactions
-----------------------------
+Relay and Mining: Priority transactions
+---------------------------------------
 
 Transactions that do not pay the minimum relay fee, are called "free
-transactions" or priority transactions. Previous versions of Bitcoin
-Core would relay and mine priority transactions depending on their
-setting of `-limitfreerelay=<r>` (default: `r=15` kB per minute) and
-`-blockprioritysize=<s>` (default: `50000` bytes of a block's
-priority space).
-
-Priority code is planned to get moved out of from Bitcoin Core 0.13
-and the default block priority size has been set to `0` in Bitcoin Core
-0.12.
+transactions" or priority transactions. Bitcoin Core relays and mines
+priority transactions depending on the setting of `-limitfreerelay=<r>`
+(default: `r=15` kB per minute) and `-blockprioritysize=<s>`.
+
+In Bitcoin Core 0.12, priority transactions are not accepted to the mempool nor
+relayed if mempool limiting has triggered a higher effective minimum relay fee.
+
+Mining of priority transactions is now disabled by default. To re-enable it,
+simply set `-blockprioritysize=<n>` where <n> is the size in bytes of your
+blocks to reserve for priority transactions. The old default was 50k, so to
+retain the same policy, you must set `-blockprioritysize=50000`; however,
+miners are encouraged to consider the trade-off between network health and
+fees, and make their own decision for how much of the block to commit for this
+purpose.
+
+Additionally, calculation of the priority for transactions is no longer updated
+correctly when they are received with unconfirmed inputs. There is a proposed
+fix for this (#7149), but due to lack of review, it is not ready in time for
+0.12.0. Miners concerned with this regression should consider merging the fix
+themselves.
```

Comment:
Users should be doing their own policy patching anyway. Core is a *reference* codebase, it isn't meant to be just taken and used as-is, especially for miners.

Justifying a clear bug by "intentional change" is just silly, especially when the new behaviour is not even deterministic. Note that the comments in #7008 do not in fact discuss this matter... and much of the support for merging the broken behaviour without the fix came from a misunderstanding that it was expensive to fix (which it isn't).